lib/services/profiles-api-service.js

```diff
@@ -213,6 +215,14 @@ function profileApiServiceFactory(
             });
         })
         .then(function () {
+            if (options.type === 'switch') {
+                return waterline.nodes.updateByIdentifier(
+                    node.id,
+                    { snmpSettings: { host: ip, community: community} }
```

Comment:
I'm not sure there's any guarantee that these settings will work though. The switch config could not enable snmp or could specify a different community. It seems more versatile to me to expose this functionality as a task that just calls `updateByIdentifier`, but at least in that case a user can change the settings via the API rather than them being hardcoded.